### ISSUE TITLE
multi gpu learning

### ISSUE THREAD

**Issue by minjuprinces1004** [AlexeyAB/darknet#1486]
hi, alexey.

some differences occur when learning the same learning data in multi-gpu under the same conditions.
for example, when learning from single-gpu, images that are detected may not be detected in multi-gpu.
when i look at the train_detector function of detector.c, the learning_rate value of each network is initialized differently.
===================================================================
nets[i]->learning_rate *= ngpus;

why they set different learning_rate for each network?

thank you alexey, always.

**Comment by fengxiuyaun** [comment]
when learning_rate =0.1，number of GPU is 4, each GPUs's learning rate=0.1*4

**Comment by AlexeyAB** [comment]
@minjuprinces1004 Hi,

More about it: https://github.com/AlexeyAB/darknet#how-to-train-with-multi-gpu

Also: https://github.com/AlexeyAB/darknet/issues/1165#issuecomment-421334900
